frontend/packages/dev-console/integration-tests/views/topology.view.ts

```diff
@@ -72,10 +76,131 @@ export const valueField = element.all(by.css('[placeholder="value"]')).last();
 // Connection lines displayed in Topology view
 export const topologyConnectors = element.all(by.css('[data-test-id="edge-handler"]'));
 
-// Elements in the topology controlbar - TBD
+// empty topology form
+export const emptyTopologyView = element(by.css('div.loading-box.loading-box__loaded'));
+
+export const listViewObj = {
+  workloadSection: element(by.css('div.co-m-pane')),
+  firstAppName: element.all(by.css('div.list-group-item')).get(0),
+  appNames: element.all(by.css('span.co-m-resource-icon.co-m-resource-deployment')),
+  switchToToplogyView: element(by.css('a.pf-c-button.pf-m-plain')),
+  deleteText: element(by.css('h3.project-overview__item-heading span.co-resource-item__deleting')),
+};
+
+export const topologyViewObj = {
+  appCount: element.all(by.css('g.odc-base-node')),
+  switchToListView: element(by.css('a.pf-c-button.pf-m-plain')),
+};
 
 // Navigate to topology tab
 export const navigateTopology = async function() {
   await browser.wait(until.elementToBeClickable(topologyNavigate));
   await topologyNavigate.click();
 };
+
+export const sideBarObj = {
+  sideBar: element(by.css('div.pf-topology-side-bar__body')),
+  Title: element(by.css('h1.co-m-pane__heading')),
+  ActionsMenu: element(by.css('[data-test-id="actions-menu-button"]')),
+};
+
+export const deleteDeployPopupObj = {
+  form: element(by.css('form.modal-content')),
+  checkbox: element(by.css('input[type="checkbox"]')),
+  cancel: element(by.css('[data-test-id="modal-cancel-action"]')),
+  delete: element(by.css('#confirm-action')),
+};
+
+export enum Actions {
```

Comment:
These aren't generic actions but rather deployment actions.
```suggestion
export enum DeploymentActions {
```

Reply:
Updated the enum Name to DeploymentActions and also updated their references